src/lib/rest/StringFilter.cpp

```diff
@@ -458,6 +458,58 @@ bool StringFilterItem::valueGet
 
 /* ****************************************************************************
 *
+* opFind - 
+*/
+static StringFilterOp opFind(char* s, char** restPP)
```

Comment:
Hard to understand parameters... An explatation in the function prebable would be great.